posts/2021-10-05-beta-release-post.adoc

```diff
@@ -0,0 +1,174 @@
+                         headersRef="myHeadersID">
+    </httpEndpoint> 
+     
+     <headers id="myHeadersID">
+            <add>foo:bar</add>
+            <add>foo:bar2</add>
+            <set>customHeader:customValue</set>
+            <setIfMissing>X-Forwarded-Proto:https</setIfMissing>
+            <remove>Via</remove>
+     </headers>  
+----
+
+ The `add` attribute allows multiple headers with the same name to be added to a response, similar to the HttpServletResponse's `addHeader` API. Similarly, the `set` attribute is analogous to the `setHeader` API, which sets a response header to the given name and value. This overwrites existing headers that share the same name. The `setIfMissing` attribute will only set the configured headers if they are not already present on the response. Lastly, the remove option will remove any response headers whose name matches a name defined by the configuration.
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>20.0.0.11-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[20.0.0.11-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+As documented in the release of link:{url-prefix}/blog/2021/07/12/logging-format-21008-beta.html#JEE9_1[Open Liberty 21.0.0.8-beta], the convenience features `jakartaee-9.0`, `jakarteeClient-9.0` and `webProfile-9.0` have now been deleted in Open Liberty 21.0.0.11-beta, and completely replaced by `jakartaee-9.1`, `jakarteeClient-9.1` and `webProfile-9.1`.
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>jakartaee-9.1</feature>
+  </featureManager>
+----
+
+Or you can add the Web Profile convenience feature to enable all of the Jakarta EE 9 Web Profile beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>webProfile-9.1</feature>
+  </featureManager>
+----
+
+=== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
```

Comment:
```suggestion
To try out these Jakarta EE 9.1 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9.1 Beta Features package instead of the main release. The beta works with Java SE 17, Java SE 11, or Java SE 8.
```